basic/dapps/ethereum-provider/.codesandbox/tasks.json

```diff
@@ -0,0 +1,31 @@
+{
+  // These tasks will run in order when initializing your CodeSandbox project.
+  "setupTasks": [
+    {
+      "name": "Install Dependencies",
+      "command": "yarn install"
+    }
+  ],
+
+  // These tasks can be run from CodeSandbox. Running one will open a log in the app.
```

Comment:
Are these auto generated bc generally comments in JSON are invalid as shown here 🤔

Reply:
Yep autogenerated by codesandbox. Ran locally and seems to run properly w/o errors